userspace/libsinsp/container_engine/mesos.cpp

```diff
@@ -24,9 +24,9 @@ limitations under the License.
 
 bool libsinsp::container_engine::mesos::match(sinsp_threadinfo* tinfo, sinsp_container_info &container_info)
 {
-	for(auto it = tinfo->m_cgroups.begin(); it != tinfo->m_cgroups.end(); ++it)
+	for(const auto& it : tinfo->cgroups())
 	{
-		string cgroup = it->second;
+		string cgroup = it.second;
```

Comment:
Question (also applies below): do we have a good reason to have `string cgroup` and not `const string& cgroup`?

Reply:
Haven't checked, honestly. Usually I prefer to limit the scope of my changes, but I can check and address it eventually.